### DISCUSSION TITLE
Quick help on a probably stupid question

### DISCUSSION THREAD

**Discussion by PJBLinkHA** [zachowj/node-red-contrib-home-assistant-websocket#633] [Q&A] [answered]
Hi everyone, 

It will probably seem obvious to everyone but I tried a lot of different things with no luck...

What should I put here, to disable that time node that I made visible from node red in HA ?
![image](https://user-images.githubusercontent.com/101043613/166978660-861e788d-2d51-4081-a06f-65c5907ed000.png)

I works perfectly if I just create a switch on my dashboard, but I'm unable to get it to work in an automation .....

Thanks in advance for you help!

**Comment by zachowj** [comment, marked as answer]
Nodes exposed to HA are just normal switches in HA. Use the service `switch.turn_off`

**Reply by PJBLinkHA** [reply]
Gosh I knew that was stupid....
Thanks a lot, works like a charm.

For further use, where do I find information on the data I can use in this field?

Thanks a lot, 
Best,

**Reply by zachowj** [reply]
https://zachowj.github.io/node-red-contrib-home-assistant-websocket/guide/custom_integration/event-nodes.html
